homeassistant/components/wink.py

```diff
@@ -135,7 +136,7 @@ def _pubnub_update(self, message):
     @property
     def unique_id(self):
         """Return the ID of this Wink device."""
-        return '{}.{}'.format(self.__class__, self.wink.device_id())
+        return '{}.{}'.format(self.__class__, self.name)
```

Comment:
I think you can remove this property if you device have no real unique ID. I see no sense to use a name on this place.